server/server.js

```diff
@@ -4,6 +4,9 @@
   Module imports
 \******************************************************************************/
 
+// Import variables from .env file.
+require('dotenv').config()
```

Comment:
Unfortunately, This is required to get around a bug in vscode. Currently the editor fails to set environment variables when running in the linux subsystem.

The good thing is that dotenv is a tiny dependency-less module which just silently does its thing. It will always take lower priority under variables defined before variables defined in `.env`, and won't error out if `.env` isn't found.